test/generators/serializer_generator_test.rb

```diff
@@ -46,7 +46,11 @@ def test_generates_attributes_and_associations
   def test_with_no_attributes_does_not_add_extra_space
     run_generator ["account"]
     assert_file "app/serializers/account_serializer.rb" do |content|
-      assert_no_match /\n\nend/, content
+      if RUBY_PLATFORM =~ /mingw/
+        assert_no_match /\r\n\r\nend/, content
+      else
+        assert_no_match /\n\nend/, content
+      end
```

Comment:
Can not write like `assert_no_match /\r?\n\r?\nend/, content`, because will cause below error.

```
  1) Failure:
SerializerGeneratorTest#test_with_no_attributes_does_not_add_extra_space [c:/git/emberjs/active_model_serializers/test/g
enerators/serializer_generator_test.rb:48]:
Expected /\r?\n\r?\nend/ to not match "\nclass AccountSerializer < ActiveModel::Serializer\n  attributes :id\n\nend\n\n"
```